### PR TITLE
Added troposphere.io weather API

### PR DESCRIPTION
**Pull request by ManuelBauer**
The troposphere.io weather API provides free (up to 1,000 free api calls per month) access to weather and climate data.

- [x] My submission is formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

**Comment by ManuelBauer** [comment]
Any updates on the pull request status? Would be nice to be a part of this list :)

File: README.md [hunk 1]
```diff
@@ -920,6 +920,7 @@ API | Description | Auth | HTTPS | CORS |
 | [OpenUV](https://www.openuv.io) | Real-time UV Index Forecast | `apiKey` | Yes | Unknown |
 | [OpenWeatherMap](https://openweathermap.org/api) | Weather | `apiKey` | Yes | Unknown |
 | [Storm Glass](https://stormglass.io/) | Global marine weather from multiple sources | `apiKey` | Yes | Yes |
+| [Troposphere.io](https://www.troposphere.io/developer) | Global weather and climate data | `apiKey` | Yes | Yes |
```
Comment (PawelBorkar) [inline README.md:923]:
```suggestion
| [Troposphere](https://www.troposphere.io/developer) | Global weather and climate data | `apiKey` | Yes | Yes |
```

Comment (PawelBorkar) [reply README.md:923]: No need to add the TLD as it was already there in the [URL](https://www.troposphere.io/developer)

**Review by PawelBorkar — CHANGES_REQUESTED** [review]
Please do the change stated below and also resolve the conflict.

**Comment by ManuelBauer** [comment]
@PawelBorkar I've fixed the issue with the TLD and resolved the conflict :)

**Comment by PawelBorkar** [comment]
Thanks @ManuelBauer!
Can you please squash the commits into a single one?

Otherwise you can also make a new PR and ref. this whatever seems comfortable to you.

**Comment by ManuelBauer** [comment]
@PawelBorkar I've squashed all commits into a single one 👍

**Comment by PawelBorkar** [comment]
Thanks for your contribution @ManuelBauer ☺️

**Pull request merged**